### PR TITLE
Added PastebinBee

### PR DESCRIPTION
**Pull request by seblw**
#30

### CODE REVIEW

File: bees/pastebinbee/pastebinbee.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		exposure := ""
+
+		action.Options.Bind("title", &title)
+		action.Options.Bind("content", &content)
+		action.Options.Bind("expire", &expire)
+		action.Options.Bind("exposure", &exposure)
+
+		ret, err := mod.client.PasteAnonymous(content, title, "text", expire, exposure)
+		if err != nil {
+			mod.LogErrorf("Error occured during posting to Pastebin: ", err)
```
Comment (muesli) [inline bees/pastebinbee/pastebinbee.go]: It's "occurred" and I'd add a "%v" to make the LogErrorf print the error properly.

File: bees/pastebinbee/pastebinbee.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+		} else {
+			mod.Logln("Paste URL: ", ret)
```
Comment (muesli) [inline bees/pastebinbee/pastebinbee.go]: No trailing space needed. Logln will prepend that for each parameter.

File: bees/pastebinbee/pastebinbee.go (hunk 1 above)
Comment (seblw) [inline bees/pastebinbee/pastebinbee.go]: ok, thanks for hint!

File: bees/pastebinbee/pastebinbeefactory.go [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+}
+
+// Image returns the filename of an image for this Bee.
+func (factory *PastebinBeeFactory) Image() string {
+	return factory.ID() + ".png"
+}
+
+// LogoColor returns the preferred logo background color (used by the admin interface).
+func (factory *PastebinBeeFactory) LogoColor() string {
+	return "#003b66"
```
Comment (muesli) [inline bees/pastebinbee/pastebinbeefactory.go]: Pick your own color code. Ideally, we don't rely on dupes to soon.

File: bees/pastebinbee/pastebinbee.go [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+// Run executes the Bee's event loop.
+func (mod *PastebinBee) Run(eventChan chan bees.Event) {
+	mod.client = pastebin.NewPastebin(mod.apiDevKey)
+}
+
+// ReloadOptions parses the config options and initializes the Bee.
+func (mod *PastebinBee) ReloadOptions(options bees.BeeOptions) {
+	mod.SetOptions(options)
+
+	options.Bind("apiDevKey", &mod.apiDevKey)
```
Comment (muesli) [inline bees/pastebinbee/pastebinbee.go]: Let's stick to JSON style here: api_dev_key.

File: bees/pastebinbee/pastebinbeefactory.go [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+// LogoColor returns the preferred logo background color (used by the admin interface).
+func (factory *PastebinBeeFactory) LogoColor() string {
+	return "#00796B"
+}
+
+// Options returns the options available to configure this Bee.
+func (factory *PastebinBeeFactory) Options() []bees.BeeOptionDescriptor {
+	opts := []bees.BeeOptionDescriptor{
+		{
+			Name:        "apiDevKey",
```
Comment (muesli) [inline bees/pastebinbee/pastebinbeefactory.go]: Let's stick to JSON style here: api_dev_key.

**Pull request merged**